user/migrations/0001_initial.py

```diff
@@ -7,7 +7,6 @@
 
 
 class Migration(migrations.Migration):
-
     initial = True
```

Comment:
Do not introduce changes to the already created migrations, it relates to the `cinema` app as well.